data/_models/shared/Processes.csv

```diff
@@ -1,119 +1,143 @@
+Variant interpretations,,classification summary,text,,,,,,,,,http://purl.obolibrary.org/obo/NCIT_C86073,Context or an explanation on how the variant classification was derived,Patient registry
+Variant interpretations,,inheritance,ontology,,,CatalogueOntologies,Inheritance pattern,,,,,http://purl.obolibrary.org/obo/GENO_0000141,"The pattern in which a genetic trait or condition is passed from one generation to the next, as determined by genetic interactions between alleles of the causal gene, and interactions between these alleles and the environment.",Patient registry
+Variant interpretations,,disease,ontology,,,CatalogueOntologies,Diseases,,,,,http://purl.obolibrary.org/obo/NCIT_C15607,E.g. Spinocerebellar ataxia type 34,Patient registry
+Variant interpretations,,category,ontology,,,CatalogueOntologies,Variation category,,,,,http://semanticscience.org/resource/SIO_001331,"The diagnosis assigned for this particular observation (e.g., ""Spinocerebellar ataxia type 34"")",Patient registry
+Variant interpretations,,consensus,refback,,,,Interpretation consensus,,variant interpretation,,,http://purl.obolibrary.org/obo/NCIT_C64197,Agreement as the result of multiple results or opinions on genomic variants,Patient registry
+Interpretation consensus,Analyses,,,,,,,,,,,http://purl.obolibrary.org/obo/NCIT_C64197,Agreement as the result of multiple results or opinions on genomic variants,Patient registry
+Interpretation consensus,,variant interpretation,ref,,,,Variant interpretations,,,,,http://purl.obolibrary.org/obo/NCIT_C164813,"If applicable, a reference to one or more variants involved in the consensus process",Patient registry
+Interpretation consensus,,name of commenter,ref,,,,Contacts,,,,,http://purl.obolibrary.org/obo/NCIT_C25461,The name of the person making the comment,Patient registry
+Interpretation consensus,,classification,ontology,,,CatalogueOntologies,Variation clinical relevance,,,,,http://edamontology.org/operation_3225,"Classification of the clinical relevance of the variant (e.g., ""Pathogenic"")",Patient registry
+Interpretation consensus,,summary,text,,,,,,,,,http://purl.obolibrary.org/obo/NCIT_C86073,Context or an explanation on how the variant classification was derived,Patient registry
+Interpretation consensus,,timestamp,datetime,,,,,,,,timestamp === null ? new Date().toISOString() : timestamp,http://purl.obolibrary.org/obo/NCIT_C54086,The date and time a comment was made,Patient registry
+Individual consent,Processes,,,,,,,,,,,http://purl.obolibrary.org/obo/NCIT_C16735,"Consent given by a patient to a surgical or medical procedure or participation in a study, examination or analysis after achieving an understanding of the relevant medical facts and the risks involved.","Patient registry, FAIR Genomes"
```

Comment:
For me, consent is more a 'material' than a 'process'. I however expect us to merge all these hierarchies at some point.

Reply:
indeed. Offtopic for now. I will put it note in epic which I now remember is actually https://github.com/molgenis/GCC/issues/1084